server/routes/register.js

```diff
@@ -2,23 +2,24 @@ const bcrypt = require("bcrypt");
 const express = require("express");
 const jwt = require("jsonwebtoken");
 const router = express.Router();
-
 const User = require("../models/user");
+const validateBody = require('../middleware/validateBody');
+const validateEntryReq = validateBody.entry;
 
-router.post("/", async function(req, res) {
-  const {email, password} = req.body;
+router.post("/", validateEntryReq, async function(req, res) {
+  const {email, password, courses, university} = req.body;
   const hashedPw = await bcrypt.hash(password, 10)
   .catch(() => {
-        res.status(500).send();
+        res.sendStatus(500).send();
   });
   if (hashedPw) {
-    const user = new User({ email, password: hashedPw });
+    const user = new User({ email, password: hashedPw, courses, university });
     const userDoc = await user.save()
     .catch((err) => {
       if (!err.errors || err.errors.email && err.errors.email.reason) {
-        res.status(500).send(); //Internal error connecting with MongoDB 
+        res.sendStatus(500); //Internal error connecting with MongoDB 
       } else {
-        res.send(409).send(); //Email conflict with existing user
+        res.sendStatus(409); //Email conflict with existing user
```

Comment:
Send a message along with the response just so we clarify what the conflict was (duplicate email)